src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs

```diff
@@ -27,6 +29,20 @@
           (style/create-link #(on-back)
             (icons/font-icon {:style {:fontSize "70%" :marginRight "0.5em"}} :angle-left)
             "Back to table")
+          [:div {:style {:marginTop "1em"}}
+           (style/create-section-header (str "Selected Configuration : "
```

Comment:
I don't quite understand what you mean here.  Do you mean it should be as "SelectedConfiguration : " (space between selected and configuration removed)  or "Selected Configuration: " (space between configuration and colon remove)?  OR do you mean there should only be one word here?